Makefile

```diff
@@ -187,8 +187,9 @@ delete-cost-models:
 
 delete-test-customer-data: delete-test-sources delete-cost-models
 
+TEST_SOURCE=all
 load-test-customer-data:
-	$(SCRIPTDIR)/load_test_customer_data.sh $(start) $(end)
+	$(SCRIPTDIR)/load_test_customer_data.sh $(TEST_SOURCE) $(start) $(end)
```

Comment:
This is a nit (so not a blocker) but it would be nice to update the make help for this command to show that you can pass in `TEST_SOURCE` if you want to just create aws/azure/etc

Reply:
Good point and nice catch